src/Microsoft.AspNet.Mvc.ViewFeatures/Rendering/HtmlHelperFormExtensions.cs

```diff
@@ -308,6 +394,7 @@ public static MvcForm BeginRouteForm(this IHtmlHelper htmlHelper, object routeVa
                 routeName: null,
                 routeValues: routeValues,
                 method: FormMethod.Post,
+                antiforgery: false,
```

Comment:
Confusion zone. Why is the `BeginRouteForm()` parameter `antiforgery` while the `BeginForm()` parameter is `suppressAntiforgery`?